crates/diesel_models/src/dynamic_routing_stats.rs

```diff
@@ -0,0 +1,77 @@
+use diesel::{Insertable, Queryable, Selectable};
+use serde::{Deserialize, Serialize};
+
+use crate::schema::dynamic_routing_stats;
+
+#[derive(Clone, Debug, Eq, Insertable, PartialEq, Serialize, Deserialize)]
+#[diesel(table_name = dynamic_routing_stats)]
+pub struct DynamicRoutingStatsNew {
+    pub payment_id: String,
+    pub tenant_id: String,
+    pub merchant_id: String,
+    pub profile_id: String,
+    pub success_based_routing_connector: String,
+    pub payment_connector: String,
+    pub currency: Option<String>,
+    pub payment_method: Option<String>,
+    pub capture_method: Option<String>,
+    pub authentication_type: Option<String>,
+    pub payment_status: String,
```

Comment:
This table is temprory until we shift to clickhouse (after which it will probably be dropped), so for quick and ease of dev it was made as string, moreover this table is only populated once afterwards there won't be any operations(CRUD) performed, this will be only accessed from grafana. So it was made like this, what you suggest?

Reply:
This is done as-well.